bin/handler-opsgenie.rb

```diff
@@ -58,6 +62,7 @@ def create_alert(description)
     tags << 'unknown' if event_status >= 3
     tags << 'critical' if event_status == 2
     tags << 'warning' if event_status == 1
+    event_tags.each { |tag, value| tags << "#{tag}_#{value}" }
```

Comment:
I think tags are optional and hence can be nil, so can you make this `event_tags.try(:each){ |tag, value| tags << "#{tag}_#{value}" }` and test if this works in both cases as expected?